modules/exploits/multi/http/h2_alias.rb

```diff
@@ -0,0 +1,79 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'H2 Arbitrary Code Execution using CREATE ALIAS',
```

Comment:
I can see `h2` being a product naming collision in the future, maybe `H2 Database CREATE ALIAS RCE` is more descriptive but shorter?

Reply:
Agreed. The original exploit name was similar; `H2 Database 'Alias' Arbitrary Code Execution`.